omnibus/config/software/datadog-agent-integrations.rb

```diff
@@ -73,11 +101,10 @@
 
     # Windows pip workaround to support globs
     python_bin = "\"#{windows_safe_path(install_dir)}\\embedded\\python.exe\""
-    python_pip = "\"import pip, glob; pip.main(['install', '-c', '#{install_dir}/agent_requirements.txt'] + glob.glob('*.whl'))\""
+    python_pip = "pip install -c #{windows_safe_path(install_dir)}\\agent_requirements.txt #{windows_safe_path(project_dir)}"
 
     if windows?
-      pip "wheel --no-deps .", :cwd => "#{project_dir}/datadog_checks_base"
-      command("#{python_bin} -c #{python_pip}", cwd: "#{project_dir}/datadog_checks_base")
+      command("#{python_bin} -m #{python_pip}\\datadog_checks_base")
```

Comment:
why has this changed? And why for windows only?

Reply:
This started breaking on windows after the bump to `pip` 10, it never complained on linux so I didn't consolidate.